src/test/java/org/reflections/Issue269LongWay.java

```diff
@@ -0,0 +1,43 @@
+package org.reflections;
+
+import java.lang.annotation.*;
+import org.junit.*;
+import java.lang.reflect.*;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.reflections.scanners.*;
+import org.reflections.util.*;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static org.junit.Assert.*;
+
+@Retention(RUNTIME)
+@interface Top {}
+
+@Top
+@Retention(RUNTIME)
+@interface Mid {}
+
+@Mid
+@Retention(RUNTIME)
+@interface Down {}
+
+class C {
+    @Down public void inC() {}
+}
+
+public class Issue269LongWay {
+    @Test
+    public void testIssue269LongWay() throws NoSuchMethodException {
```

Comment:
I think it's better to give the test case(and file) a meaningful name, so that people will know what the class or file is for at the first glance.